pages/common/npx.md

```diff
@@ -3,21 +3,21 @@
 > Execute binaries from `npm` packages.
 > More information: <https://github.com/npm/npx>.
 
-- Execute the binary from a given npm module:
+- Execute the command from a local or remote `npm` package:
 
-`npx {{module_name}} {{command_arguments}}`
+`npx {{command}} {{command_arguments}}`
```

Comment:
This should be the package name, e.g. `react-native-cli`, not the executable `react-native`
```suggestion
`npx {{package}} {{command_arguments}}`
```

Reply:
Actually there's good overlapping between the two since, most of the time, commands have the same name of packages.
Because of that is both possible to call `npx react-native-cli` or `npx react-native`.

I could suggest to use a syntax that can include both as sometimes is shown in the man pages: ```npx {{package|command}} {{command_arguments}}```

What do you think about that?

Reply:
After all the result of `man npx` is
```
[....]

 npx <pkg>[@<specifier>] [args...]
 npx -p <pkg>[@<specifier>] <cmd> [args...]
 npx -c '<cmd> [args...]'
 npx -p <pkg>[@<specifier>] -c '<cmd> [args...]'

[...]        ```

Reply:
@ffranchina `{{package|command}}` sounds good to me!